src/common/KeyboardAvoider.js

```diff
@@ -20,7 +39,13 @@ type Props = $ReadOnly<{|
  */
 export default class KeyboardAvoider extends PureComponent<Props> {
   render() {
-    const { behavior, children, style, contentContainerStyle } = this.props;
+    const {
+      behavior,
+      children,
+      style,
+      contentContainerStyle,
+      keyboardVerticalOffset = 0,
```

Comment:
Does this need a default here? Docs say it has the same value as a default at the layer below:
https://reactnative.dev/docs/keyboardavoidingview#keyboardverticaloffset

Seeing this, I was wondering if by starting to pass 0 here we were changing any behavior. It looks like the answer is no but I think it'd be easier to see that if it's just another prop we pass straight through.